plugins/livereload/mod.ts

```diff
@@ -0,0 +1 @@
+export { default } from 'https://deno.land/x/drollup_plugin_livereload/mod.ts'
```

Comment:
I was thinking... perhaps instead of having `deno-rollup` act as a barrel file / re-exporter for your plugin(s) perhaps we should instead just link to your repo directly - otherwise whenever / if you ever update your plugin(s) we will have to also go through and update the READMEs etc. here to keep the information in sync? IMO a single source of truth of your repo makes sense, but interested to know what you think?

What we _could_ do is link to your plugin repo(s) in the table on the plugins README page ( https://github.com/cmorten/deno-rollup/blob/main/plugins/README.md ), and then have examples here as well ( as your PR description suggests you will ) which I'm not so concerned about becoming outdated ( so long as we fix the version the examples use so they continue to work and don't automatically inherit breaking changes! ).